### ISSUE TITLE
detail labelling of GMO cultures

### ISSUE THREAD

**Issue by ilevantis** [london-biohackers/biohackspace-sops#15]
using red + green dot stickers

**Comment by AJamesPhillips** [comment]
Sounds good.  So we'd put these on petri dishes, and 2, 15, 50 ml conical tubes, and flasks?

**Comment by ilevantis** [comment]
yeah anything with cells growing in it: Green dot sticker for non-GM, Red dot sticker for GM.
